Remotion/Data/DomainObjects.UnitTests/Mapping/TestMappingConfiguration.cs

```diff
@@ -60,6 +60,9 @@ public static TestMappingConfiguration Instance
 
     public static void Initialize ()
```

Comment:
Rename: `EnsureInitialized` -> makes it obvious on the callsite that this is allowed to be called multiple times.
(also update the exception message in line 55)